server/tokens.go

```diff
@@ -18,6 +18,11 @@ type blindedTokenIssueRequest struct {
 	BlindedTokens []*crypto.BlindedToken `json:"blinded_tokens"`
 }
 
+type newBlindedTokenIssueRequest struct {
```

Comment:
Instead of `new` what do you think of naming this `v2` here and elsewhere?

Reply:
Yep, makes sense